README.md

```diff
@@ -4,8 +4,8 @@
 env name|required|default|description
 --------|--------|-------|-----------
 DATALAKE_CHANNEL_ID|True|None|DataLake channel id
-DATALAKE_TRAIN_FILE_ID|True|None|DataLake file id
-DATALAKE_TEST_FILE_ID|False|None|DataLake file id
+DATALAKE_TRAIN_FILE_ID|True|None|DataLake file id for training
+DATALAKE_VAL_FILE_ID|False|None|DataLake file id for validating
```

Comment:
Rename `test` to `val` like [object detection template](https://github.com/abeja-inc/platform-template-object-detection/blob/master/dataset.py#L17).